ignite/distributed/auto.py

```diff
@@ -341,3 +341,12 @@ def __init__(self, optimizer: Optimizer) -> None:
 
         def step(self, closure: Optional[Callable] = None) -> None:
             xm.optimizer_step(self.wrapped_optimizer, barrier=True)
+
+
+def autocast(enabled: bool) -> Any:
```

Comment:
I do not quite understand why we have to introduce a public method for testing. This not a good practice, IMO. Let's remove it.